src/components/InteractionLayer/index.js

```diff
@@ -106,50 +104,17 @@ class InteractionLayer extends React.Component {
         },
       });
     }
-    if (onMouseMove) {
-      const xScale = createXScale(subDomain, width);
-      const rawTimestamp = xScale.invert(xpos).getTime();
-      const output = { xpos, ypos, points: [] };
-      series.forEach(s => {
-        const { data, xAccessor, yAccessor, yDomain } = s;
-        const rawX = d3.bisector(xAccessor).left(data, rawTimestamp, 1);
-        const x0 = data[rawX - 1];
-        const x1 = data[rawX];
-        let d = null;
-        if (x0 && !x1) {
-          d = x0;
-        } else if (x1 && !x0) {
-          d = x1;
-        } else if (!x0 && !x1) {
-          d = null;
-        } else {
-          d =
-            rawTimestamp - xAccessor(x0) > xAccessor(x1) - rawTimestamp
-              ? x1
-              : x0;
-        }
-        if (d) {
-          const yScale = createYScale(yDomain, height);
-          const ts = xAccessor(d);
-          const value = yAccessor(d);
-          output.points.push({
-            id: s.id,
-            timestamp: ts,
-            value,
-            x: xScale(ts),
-            y: yScale(value),
-          });
-        } else {
-          output.points.push({ id: s.id });
-        }
+    if (onMouseMove || ruler) {
```

Comment:
This fixes #24 -- don't forget to put that in your commit!

Reply:
Good catch